www/docs/providers/vk.md

```diff
@@ -0,0 +1,25 @@
+---
+id: vk
+title: vk.com
+---
+
+## Documentation
+
+https://vk.com/dev/first_guide
+
+## Configuration
+
+https://vk.com/apps?act=manage
+
```

Comment:
Do explain here the possibility to use different API versions, and that `5.126` is the default. Maybe add a link to the corresponding vk.com page as well.